terraform/securityhub.tf

```diff
@@ -0,0 +1,8 @@
+#####################################
+# Security Hub for AWS root account #
+#####################################
```

Comment:
Does this need to be here?

Reply:
There's going to be similar for each region & organisation-security, so figured it keeps it separated and a bit easier to read.